backend/btrixcloud/orgs.py

```diff
@@ -480,8 +480,11 @@ async def remove_user_from_org(
                 raise HTTPException(
                     status_code=400, detail="Can't remove only owner from org"
                 )
+        try:
+            del org.users[str(other_user.id)]
+        except KeyError:
+            raise HTTPException(status_code=404, "no_such_org_user")
```

Comment:
```suggestion
raise HTTPException(status_code=404, detail="no_such_org_user")
```
To fix linting failure.